### PR TITLE
Feature/gui-input-expression-style

### PR DESCRIPTION
**Pull request by w-disaster**
- Substitute `TextArea` with `TextPane`
- Styled operators inside input `TextPane`

### CODE REVIEW

File: src/main/scala/satify/view/ComponentUtils.scala [hunk 1]
```diff
@@ -30,24 +35,73 @@ object ComponentUtils:
...
+    updateStyle(textPane)
+    textPane.reactions += { case ValueChanged(t: TextPane) =>
+      val s = t.text
+      if s != textPaneText then updateStyle(t)
+    }
+    textPane
 
   def createParameterInputText(): TextField =
-    new TextField:
+    val tf = new TextField:
```
Comment (Mala1180) [inline src/main/scala/satify/view/ComponentUtils.scala:98]: Why did you insert val?

File: src/main/scala/satify/view/ComponentUtils.scala [hunk 2]
```diff
@@ -30,24 +35,73 @@ object ComponentUtils:
...
+      textPaneText = text
+
+      // Util function to minimize repetitions
+      def f(h: Int => Int, from: Int, length: Int): List[Int] =
+        val start = h(from)
+        if start != -1 then f(h, from + length, length) :+ start
+        else Nil
+
+      // Set character attribute foreach offset inside the input list
+      def g(l: List[Int], length: Int): Unit = Swing.onEDT {
```
Comment (Mala1180) [inline src/main/scala/satify/view/ComponentUtils.scala:58]: I would change names of `f` `g` and `h`

**Review by Mala1180 — CHANGES_REQUESTED** [review]

File: src/main/scala/satify/view/ComponentUtils.scala (hunk 1 above)
Comment (w-disaster) [reply src/main/scala/satify/view/ComponentUtils.scala:98]: That's a typo, thanks for reporting it

File: src/main/scala/satify/view/ComponentUtils.scala (hunk 2 above)
Comment (w-disaster) [reply src/main/scala/satify/view/ComponentUtils.scala:58]: Alright!

**Review by Mala1180 — APPROVED** [review]

**Pull request merged**